image-rs/Cargo.toml

```diff
@@ -38,7 +38,7 @@ serde = { workspace = true, features = ["serde_derive", "rc"] }
 serde_json.workspace = true
 serde_yaml = { version = "0.9", optional = true }
 sha2.workspace = true
-sigstore = { git = "https://github.com/sigstore/sigstore-rs.git", rev = "69e8f33", default-features = false, optional = true}
+sigstore = { git = "https://github.com/sigstore/sigstore-rs.git", tag = "v0.7.2", default-features = false, optional = true}
```

Comment:
Yes. Seems that we have done duplicated things https://github.com/confidential-containers/guest-components/pull/360/commits/39eec751a43bbfc90c804548f75a2e5faf023ef3

but it is ok for me to get this merged first and I'll take a rebase.

Reply:
oh, I had totally missed this. There was no PR with the title indicating the overlap.

Reply:
yea. No problem. I once thought that no one care about this : )

Reply:
looks like https://github.com/confidential-containers/guest-components/pull/360/commits/fa01b0611658dbc6f1b67486840175a9c4442b6e is another overlap?

Reply:
@Xynnn007 I added signed-of-bys from both of us to the sigstore-rs commit.

Reply:
Thanks. Let's wait another maintainer to check again and get this merged.